field/composite_test.go

```diff
@@ -95,6 +95,96 @@ var (
 		},
 	}
 
+	compositeTestSpecWithDefaultBitmap = &Spec{
+		Length:      32,
+		Description: "Test Spec",
+		Pref:        prefix.ASCII.LL,
+		Bitmap: NewBitmap(&Spec{
+			Length:      8,
+			Description: "Bitmap",
+			Enc:         encoding.BytesToASCIIHex,
+			Pref:        prefix.Hex.Fixed,
```

Comment:
I think you should add `DisableAutoExpand: true` here, otherwise, if you set subfield `1`, then the packed length of the bitmap will be 16 bites. You can run an experiment to disprove this :)

But it's 100% that you don't want bitmap to auto expand like it does in the message.